tiledb/sm/filesystem/win.cc

```diff
@@ -588,7 +597,7 @@ std::string Win::path_from_uri(const std::string& uri) {
           "file:///" + uri;
 
   unsigned long path_length = MAX_PATH;
-  char path[MAX_PATH];
+  char path[INTERNET_MAX_URL_LENGTH];
```

Comment:
If it needs to be changed, then `path_length` should also use the same variable (if they are guaranteed the same value, why use different constants?; if different, then line 602 may write out of bounds)

Reply:
@ihnorton was not dangerous but not useful as I misread docs when I changed, have reverted to previous state.

Reply:
👍